sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -98,6 +102,8 @@ typedef struct _nib_onl_entry {
      * @note    Only available if @ref GNRC_IPV6_NIB_CONF_ARSM != 0.
      */
     uint8_t l2addr[GNRC_IPV6_NIB_L2ADDR_MAX_LEN];
+    evtimer_msg_event_t reach_timeout;  /**< Event for @ref GNRC_IPV6_NIB_REACH_TIMEOUT */
+    evtimer_msg_event_t delay_timeout;  /**< Event for @ref GNRC_IPV6_NIB_DELAY_TIMEOUT */
```

Comment:
Yes. And no. 14*16 isn't correct, because some go into the `nib_dr_t` and some go in the `_nib_offl_t` class. And yes. That is the caveat of having the full neighbor discovery activated (which for most nodes you don't need, because they don't need to be routers => most of the state machine is not required, according to RFC6775). This is already the centralized concept. The delta per event is handled by the event timer. This here are *only* the events (i.e. the deltas + next-pointer for the timer's queue management + message to represent the event itself in our event management [aka IPC]), not the whole event timer. You need to store these somewhere.